app/pages/sections/order-items/additional-services/routes.js

```diff
@@ -104,6 +104,12 @@ export const additionalServicesRoutes = (authProvider, addContext, sessionManage
       });
 
       if (apiResponse.success) {
+        sessionManager.clearFromSession({
+          req,
+          keys: [
+            'selectedItemId', 'selectedItemName', 'selectedRecipientId', 'selectedRecipientName', 'selectedPriceId', 'selectedCatalogueSolutionId',
+          ],
+        });
```

Comment:
Is it worth moving this into a separate function that we can reuse across additional services, associated services and catalogue solutions?

Reply:
Good shout and I was considering this also. However, I didn't want to do anything too overdo this one just yet. I think we need to discuss about the best way to handle our sessions in general and where to tackle them.

Reply:
Ah, OK. I guess we can come back to it later as tech debt then.